### PR TITLE
[stdlib] Remove FileCheck from `test_anypointer.mojo`

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
Related to #2024

### CODE REVIEW

File: stdlib/test/memory/test_unsafepointer.mojo [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ===----------------------------------------------------------------------=== #
-# RUN: %mojo %s | FileCheck %s --dump-input=always
+# RUN: %mojo -debug-level full %s
```
Comment (JoeLoser) [inline stdlib/test/memory/test_unsafepointer.mojo:13]: **Suggestion** Drop the `-debug-level full` as in other PRs.

**Review by JoeLoser — APPROVED** [review]
Great, thank you!  This one was a bit more interesting than the other ones 😄

File: stdlib/test/memory/test_unsafepointer.mojo (hunk 1 above)
Comment (gabrieldemarmiesse) [inline stdlib/test/memory/test_unsafepointer.mojo:13]:
```suggestion
# RUN: %mojo-no-debug %s
```

Comment (gabrieldemarmiesse) [reply stdlib/test/memory/test_unsafepointer.mojo]: Done

**Review by JoeLoser — APPROVED** [review]

**Pull request merged**